src/rdo.rs

```diff
@@ -771,17 +776,20 @@ pub fn rdo_mode_decision<T: Pixel>(
 
     let intra_mode_set = RAV1E_INTRA_MODES;
     let mut sads = {
+      //debug_assert!(bsize == tx_size.block_size());
       let edge_buf = {
         let rec = &ts.rec.planes[0].as_const();
         let po = tile_bo.plane_offset(&rec.plane_cfg);
-        get_intra_edges(rec, po, tx_size, fi.sequence.bit_depth, None)
+        // FIXME: If tx partition is used, get_intra_edges() should be called for each tx block
+        get_intra_edges(rec, tile_bo, 0, 0, bsize, po, tx_size, fi.sequence.bit_depth, None)
       };
       intra_mode_set
         .iter()
         .map(|&luma_mode| {
           let tile_rect = ts.tile_rect();
           let rec = &mut ts.rec.planes[0];
           let mut rec_region = rec.subregion_mut(Area::BlockStartingAt { bo: tile_bo });
+          // FIXME: If tx partition is used, luma_mode.predict_intra() should be called for each tx block
```

Comment:
is this fixme still true?

Reply:
Yes, it is about our future extension of trimming for intra predicted partition with tx split.

Reply:
Aha ok. I missed that this is in the pruning part.